docs/mmteb/points.md

```diff
@@ -0,0 +1,18 @@
+# Points
+
+| GitHub            | Total points | New dataset | New task | Dataset annotations | (Bug)fixes | Running Models | Review PR |  Paper Writing | Ideation | Coordination |
+| ----------------- | ------------ | ----------- | -------- | ------------------- | ---------- | -------------- |  -------- | -------------- | -------- | ------------- |
+| KennethEnevoldsen |              |   38        |          |                     |            |                |           |                |         |               |
+| x-tabdeveloping   |              |    2        |          |                     |            |                |           |                |        |               |
+
+Note that coordination and ideation is not included in the total points, but is used to determine first and last authors. 
+
+# Contributor Informations
+
+First name and last name are as you want them to appear in a publication.
+
+| GitHub            | First name | Last name  | Email                        | User on openreview   |
+| ----------------- | ---------- | ---------- | ---------------------------- | -------------------- |
+| KennethEnevoldsen | Kenneth    | Enevoldsen | kennethcenevoldsen@gmail.com | ~Kenneth_Enevoldsen1 |
+| x-tabdeveloping   | Márton     | Kardos     |                              | ~Márton_Kardos1      |
+
```

Comment:
Could someone help retroactively add everyone who has contributed in the past here? cc @imenelydiaker @wissam-sib